src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+
+interface Props {
+  handleAddTodo: (todoTitle: string) => Promise<void>;
+  setError: (error: string | null) => void;
+}
+
+export const TodoForm: React.FC<Props> = ({ handleAddTodo, setError }) => {
+  const [todoTitle, setTodoTitle] = useState('');
+  const [isInputDisabled, setIsInputDisabled] = useState(false);
+
+  const handleFormSubmit = async (e:React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    if (!todoTitle.trim()) {
```

Comment:
Now, regardless of this check, you can add an empty todo (but an error will appear at the bottom) and add a todo whose name will have only spaces, so the check needs to be improved a little.